sql/catalyst/src/main/java/org/apache/spark/sql/connector/catalog/constraints/Constraint.java

```diff
@@ -58,6 +58,12 @@ public interface Constraint {
    */
   String toDDL();
 
+  /**
+   * Similar to {@link #toDDL()}, but returns the description of this constraint for describing
```

Comment:
Let's make it more clear
`Returns the constraint description for DESCRIBE TABLE output, excluding the constraint name (shown separately).`